x_goal: 1
y_goal: 1
beta: 0
phi: 0